ImageGallery/src/org/sleuthkit/autopsy/imagegallery/gui/drawableviews/DrawableTileBase.java

```diff
@@ -202,7 +202,7 @@ private ContextMenu buildContextMenu(DrawableFile file) {
                 final MenuItem extractMenuItem = new MenuItem(Bundle.DrawableTileBase_menuItem_extractFiles());
                 extractMenuItem.setOnAction(actionEvent
                         -> SwingUtilities.invokeLater(() -> {
-                            TopComponent etc = WindowManager.getDefault().findTopComponent(ImageGalleryTopComponent.PREFERRED_ID);
```

Comment:
WindowManager now unused import

Reply:
I removed the unused import and updated the copyright.